modules/perception/obstacle/base/types.h

```diff
@@ -42,6 +42,12 @@ enum SensorType {
   UNKNOWN_SENSOR_TYPE = 10,
 };
 
+enum ScoreType {
+  SCORE_CNN = 0,
+  SCORE_RADAR = 1,
+  UNKNOWN_SCORE_TYPE = 10,
```

Comment:
recommend to change UNKNOWN = 0;
REF: http://androiddevblog.com/protocol-buffers-pitfall-adding-enum-values/